indico/modules/events/registration/templates/emails/base_registration.html

```diff
@@ -1,4 +1,5 @@
 {% extends 'emails/base.html' %}
+{% from 'events/emails/render_event_dates.html' import render_event_dates_html %}
```

Comment:
Usually we use an underscore-prefixed name for templates containing only macros, e.g. `_event_dates.html`